### PR TITLE
Add gift message support

### PR DESCRIPTION
**Pull request by joelind**

### CODE REVIEW

File: spec/lib/documents/shipment_order_spec.rb [hunk 1]
```diff
@@ -1,19 +1,42 @@
...
 
-      expect(subject.name).to match /^_ShipmentOrder_#{shipment['id']}_.*\.xml/
+      expect(subject.name).to match(/^_ShipmentOrder_#{shipment['id']}_.*\.xml/)
       expect(errors).to eq []
+
+      expect(xml.css('Gift')).to be_empty
+    end
+
+    context 'when shipment contains comments' do
+      before { shipment['comments'] = 'Happy Birthday' }
```
Comment (rschwager) [inline spec/lib/documents/shipment_order_spec.rb:24]: Magic string!  What happens when Quiet or Follain goes international? :)

**Review by rschwager — COMMENTED** [review]
Looks good!

**Comment by joelind** [comment]
Thanks, Russ!

**Comment by rschwager** [comment]
I'll send you my bill later.  Payment in beer is good. 

I hope all is well.

**Comment by joelind** [comment]
Works for me! Name a time and place!

**Pull request merged**